src/include/migraphx/concat_opt.hpp

```diff
@@ -9,7 +9,7 @@
 #include <utility>
 
 #include <migraphx/operation.hpp>
-#include <migraphx/operators.hpp>
+#include <migraphx/op/concat.hpp>
```

Comment:
This change needs to be update in the `concat_opt.hpp` file under the `tools/include`. And then you need to regenerate these files with `make generate`.

Reply:
fixed